packages/admin/resources/css/app.css

```diff
@@ -48,4 +48,11 @@
         z-index: 1;
         top: -1px;
     }
+
+    .filament-sidebar-header::before {
```

Comment:
I actually dislike moving code into the app.css file if it can just be added inline using Tailwind as well.